sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/PubsubSchemaCapableIOProvider.java

```diff
@@ -211,7 +216,11 @@ public POutput expand(PCollection<Row> input) {
           return input
               .apply(
                   RowToPubsubMessage.fromConfig(
-                      config, useFlatSchema, useTimestampAttribute(config)))
+                      new AutoValueSchema()
```

Comment:
Actually, I looked at the `RowToPubsubMessage` class and found 2 of the 3 parameters here (`config` and `useFlatSchema`) are not used at all. What was the reason to keep them there? Can we remove them?

Reply:
There was no reason to keep it from the old logic. Unnecessary variables removed.